client.go

```diff
@@ -285,6 +286,26 @@ func (c *Client) GetTasks(param *TasksQuery) (resp *TaskResult, err error) {
 	return resp, nil
 }
 
+func (c *Client) CancelTasks(param *TasksQuery) (resp *TaskInfo, err error) {
```

Comment:
You cannot use `TasksQuery` as a parameter here, because this method does not accept `limit` and `from`

<img width="709" alt="image" src="https://user-images.githubusercontent.com/4116980/206243750-7cfdfa07-7e0f-4ab0-b0f3-5ae18841c6b2.png">

Reply:
I also updated to **- Guarantee all the filters available in the `GET /tasks` but `from` and `limit` are available.** I hope it's clearer

Reply:
It will not send `TaskQuery` directly:
```go
	if param != nil {
		encodeTasksQuery(param, &req)
	}
	if err := c.executeRequest(req); err != nil {
		return nil, err
	}
```
But it will only encode the parameters set on the structure see `encodeTasksQuery` method

Reply:
I know that, `TaskQuery` contains two fields called `limit` and `from`, and they are not allowed in the meilisearch request `POST /tasks/cancel`. But if you share the type between this method and `get_tasks` for example, you'll support it and this is not correct :)

If you didn't get my explanation, please try to send a request to `POST /tasks/cancel` with `limit` and `from` body parameters, and see what happens, or read the spec again 😃

Reply:
I understood very well what you meant, and it is I who wrote this spec 😄
I know that the `TaskQuery` structure is every fields possible for the queries made on the `task` route, but I think that precisely the fact that the structure itself is not sent, we avoid having to create code duplication by creating three specific structures for each query.